### DISCUSSION TITLE
Cannot enable Google Calendar after 2-Step Verification turned on

### DISCUSSION THREAD

**Discussion by dncmrc** [kewisch/gdata-provider#306] [Support] [answered]
Google Calendar does not accept anymore my password after I turned on 2-Step Verification. Of course Google Calendar works fine from a web browser page. Not that I want 2-Step verification but it seems Google will turn it on anyway from Nov the 9th!
Help!

**Comment by kewisch** [comment, marked as answer]
Thanks for taking this to #308, closing out this answer.

**Reply by dncmrc** [reply]
Hi,
Really? This is still unsolved. 
Please let me know if you need any info about this issue to fix it.
Thanks,
Marco
